src/app.js

```diff
@@ -47,7 +47,7 @@ Sentry.init({
 // Build script is triggered for `npm start` and assets are served from /dist.
 // Build script is NOT run for `npm run dev`, assets are served from /src, and nodemon restarts server without build (faster dev).
 const staticPath =
-  process.env.npm_lifecycle_event === 'start' ? '../dist' : './client'
+  process.env.npm_lifecycle_event === 'start' ? 'dist' : 'src/client'
```

Comment:
Hmm, I guess this was already the case, but I'm assuming this means that these directories are relative to the directory one happens to run `npm start` from? If so, might be more resilient to make it relative to `__dirname`  (or its [ES module equivalent](https://stackoverflow.com/questions/46745014/alternative-for-dirname-in-node-js-when-using-es6-modules)) instead.

Reply:
Although I guess there are a number of places that this holds for (every relative path you had to update in this PR), so if it's too much work, consider it out of scope for this PR.

Reply:
perhaps a new ticket? 😅

Reply:
https://github.com/mozilla/blurts-server/issues/3008